style.css

```diff
@@ -0,0 +1,73 @@
+.header {
+  background-color: #3c3a39;
+  height: 48px;
+  width: 375px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.header p {
+  color: #fff;
+  font-family: 'Roboto', sans-serif;
+  margin-left: 24px;
+}
+
+.header img {
+  height: 24px;
+  width: 24px;
+  margin-right: 24px;
+}
+
+.mainContain {
+  background-color: #1c1a19;
+  height: 590px;
+  width: 375px;
+  top: 92px;
+  display: flex;
+  flex-direction: column;
+  background-image: url("images/Illustration.png");
+  background-repeat: no-repeat;
+  background-position: right bottom -45px;
+}
+
```

Comment:
- [ ] I don't think it's a good idea to specify a fixed width for your mobile version since we are using the [mobile-first approach](https://smartbear.com/learn/performance-monitoring/what-is-mobile-first/#:~:text=Mobile%20First%20Approach%20refers%20to,the%20design%20for%20mobile%20afterwards.), it means the website should be responsive for the mobile devices without having to specify the width, or the media queries, if you want to see the result in the mobile view, you can either minimize your browser window or use the developer tools in your browser to toggle the device toolbar to have a kind of phone simulator, you can learn more on how to do so [here](https://www.browserstack.com/guide/view-mobile-version-of-website-on-chrome)

Reply:
Wooow it's true now we built from small screens to big ones!! Thanks again for improving my coding skills! 🧑‍💻